FlowCrypt/App/AppStartup.swift

```diff
@@ -40,6 +40,31 @@ struct AppStartup {
         }
     }
 
+    // Update `isRevoked` value (PubKeyRealmObject `isRevoked` was added in realm schema version 10[2022 May 23th])
+    // Need to set correct `isRevoked` value when user first opens app from older versions
+    @MainActor
+    private func checkAndUpdateIsRevoked(context: AppContextWithUser) async throws {
+        let revokedUpdatedFlag = "IS_PUB_KEY_REVOKED_UPDATED"
+        let userDefaults = UserDefaults.standard
+        if userDefaults.bool(forKey: revokedUpdatedFlag) {
+           return
+        }
+        // Added storage access directly because this function logic should be removed in the near future
+        let storage = try context.encryptedStorage.storage
+        let pubKeyObjects = storage.objects(PubKeyRealmObject.self).unique()
```

Comment:
Do we need `.unique()` here?

Reply:
Yeah, in addition to get unique array, it converts Collection to Array.